packages/eds-core-react/src/components/Button/Button.stories.tsx

```diff
@@ -310,7 +325,7 @@ export const Compact: Story<ButtonProps> = () => {
       <Button variant="ghost_icon" aria-label="menu action">
         <Icon data={menu} title="Ghost icon menu"></Icon>
       </Button>
-      <Button.Group aria-label="button group compact">
+      <Button.Group aria-label="compact">
```

Comment:
Maybe a more descriptive label? "compact button group" ?

Reply:
Same in previous comment ⬆️